public/main.js

```diff
@@ -52,6 +53,8 @@ function reloadTodoList() {
         todos.forEach(function(todo) {
             var listItem = document.createElement("li");
             listItem.textContent = todo.title;
+            listItem.style.fontStyle = todo.complete === true ? "italic" : "normal";
+            listItem.style.fontWeight = todo.complete === true ? "normal" : "bold";
```

Comment:
It would be better if you added a 'complete' class and then did the styling for this in the css.